numba/npyufunc/dufunc.py

```diff
@@ -52,6 +63,10 @@ def __init__(self, py_func, **kws):
             kws.pop('identity', None))
         super(DUFunc, self).__init__(dispatcher, **kws)
         self._install_type()
+        self._kernel = type('DUFuncKernel_' + self.ufunc.__name__,
```

Comment:
I care about class and object names, especially when looking at stack traces, or debugging.